.github/workflows/build-debug-pod-image.yaml

```diff
@@ -0,0 +1,50 @@
+name: "Build Debug Pod Image"
+
+on:
+  workflow_dispatch:
+
+jobs:
+  docker:
+    name: Build and push image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
```

Comment:
Pin the binfmt image version, reference: https://github.com/GreptimeTeam/greptimedb/pull/5516